vocode/streaming/telephony/client/vonage_client.py

```diff
@@ -45,21 +45,25 @@ def create_call(
     def create_call_ncco(base_url, conversation_id, record):
         ncco = []
         if record:
-            ncco.append({
-                "action": "record",
-                "eventUrl": [f"https://{base_url}/recordings/{conversation_id}"]
-            })
-        ncco.append({
-            "action": "connect",
-            "endpoint": [
+            ncco.append(
```

Comment:
why are we making changes to the vonage client?

Reply:
Seems like this is left over from a previous commit?